pkg/storj/encryption.go

```diff
@@ -44,6 +44,9 @@ const (
 	// EncSecretBox indicates use of XSalsa20-Poly1305 encryption, as provided
 	// by the NaCl cryptography library under the name "Secretbox".
 	EncSecretBox
+	// EncURLSafeBase64 is like EncNull but Base64 encodes/decodes the
+	// binary path data (URL-safe)
+	EncURLSafeBase64
```

Comment:
Wondering whether it should be named `EncNullBase64URL` the encoding is referred to as "base64url" https://tools.ietf.org/html/rfc4648#section-5.

Alternatively `NullBase64URL` or something.

Reply:
:+1: